diagnostic.md

```diff
@@ -3,67 +3,58 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+$('OL')
+$('li')
+$('H1')
+$('Body')
```

Comment:
These are selectors but be careful about capitalization. They should be lowercase.